pyteomics/mzmlb.py

```diff
@@ -0,0 +1,445 @@
+# -*- coding: utf8 -*-
```

Comment:
This is necessary because the citation includes UTF8 characters (en-dash and the like). If you prefer, I could manually retype it using only ASCII look-alikes.

Reply:
Ah, okay. Whichever way you prefer is fine. Sorry for the noise.